tests/components/tolo/test_config_flow.py

```diff
@@ -116,3 +124,25 @@ async def test_dhcp_invalid_device(hass: HomeAssistant, toloclient: Mock) -> Non
         DOMAIN, context={"source": SOURCE_DHCP}, data=MOCK_DHCP_DATA
     )
     assert result["type"] is FlowResultType.ABORT
+
+
+async def test_reconfigure(
+    hass: HomeAssistant,
+    toloclient: Mock,
+    coordinator_toloclient: Mock,
+    config_entry: MockConfigEntry,
+) -> None:
+    """Test reconfiguration flow."""
+    result1 = await config_entry.start_reconfigure_flow(hass)
+
+    assert result1["step_id"] == "user"
+
+    toloclient().get_status.side_effect = lambda *args, **kwargs: object()
+
+    result2 = await hass.config_entries.flow.async_configure(
+        result1["flow_id"],
+        user_input={CONF_HOST: "127.0.0.1"},
+    )
+
+    assert result2["type"] is FlowResultType.ABORT
+    assert result2["reason"] == "reconfigure_successful"
```

Comment:
The way the test is written now, there's no update of the config entry. I'd suggest to parametrize the test with the new ip-address (same / different).

The test also needs to assert that the config entry data is updated.

Reply:
Fixed it, thanks for the feedback.